bartfire.py

```diff
@@ -12,6 +12,11 @@
 # Folder for debug output files
 debugFolder = "/tmp/share/debug"
 
+# Don't write to /tmp/share in cfl
+os.environ["TMPDIR"] = "/tmp"
```

Comment:
If this is where BART will write temporary files, maybe change this to /tmp/share instead, as matching start-fire-python-server.sh?